tests/test_binary.nim

```diff
@@ -20,7 +20,7 @@ template execAndCheck(expectedExitCode: int; body: untyped) {.dirty.} =
   ## `body` must have the same return type as `execCmdEx`.
   let (outp, exitCode) = body
   if exitCode != expectedExitCode:
-    echo outp
+    stderr.writeLine outp
```

Comment:
Actually I think I'll revert this one, given that all the other `unittest` output goes to stdout.